src/Reflection/ReflectionFunctionAbstract.php

```diff
@@ -257,16 +258,10 @@ public function isClosure() : bool
 
     /**
      * Is this function deprecated?
-     *
-     * Note - we cannot reflect on internal functions (as there is no PHP source
-     * code we can access. This means, at present, we can only EVER return false
-     * from this function.
-     *
-     * @see https://github.com/Roave/BetterReflection/issues/38
      */
     public function isDeprecated() : bool
     {
-        return false;
+        return strpos($this->getDocComment(), '@deprecated') !== false;
```

Comment:
Probably need to verify that `@deprecated` is followed by whitespace here

Reply:
Test case: `@deprecatedPolicy` would for example still trip this

Reply:
Since better reflection is already using phpDocumentor/ReflectionDocblock, why not use that to find the deprecation tag?

Reply:
Would most certainly help 👍

Reply:
I moved to the php documentor check. 
Ive kept the logic within the method. I could also extract the logic for checking if a tag exists to another class, but i'm not sure if it is needed. What do you think @Ocramius ?

Reply:
LGTM 👍

Just needs a  rebase now 😅